cmd/data-usage.go

```diff
@@ -51,11 +50,9 @@ const (
 
 // initDataUsageStats will start the crawler unless disabled.
 func initDataUsageStats() {
-	dataUsageEnabled, err := config.ParseBool(env.Get(dataUsageCrawlConf, config.EnableOn))
-	if err == nil && !dataUsageEnabled {
-		return
+	if env.Get(envDataUsageCrawlConf, config.EnableOn) == config.EnableOn {
```

Comment:
Technically this change breaks compatibility with the existing behavior.

Reply:
Yes it's all intentional, no one is using it yet.